app/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,7 @@
     <application
         android:name=".MifosSelfServiceApp"
         android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/ic_icon"
```

Comment:
![mifos_icon](https://cloud.githubusercontent.com/assets/16569570/23603611/e1556b24-027b-11e7-831e-4a3ea98abbb0.png)

Will this image work? @therajanmaurya